mentat/message.py

```diff
@@ -0,0 +1,53 @@
+import base64
+import json
+from enum import Enum
+from pathlib import Path
+from typing import Any
+
+import attr
+
+
+class MessageRole(Enum):
+    System = "system"
+    User = "user"
+    Assistant = "assistant"
+
+
+@attr.define
+class Message:
```

Comment:
Since none of this will be necessary once I get the OpenAI API 1.0 PR through, I'm wondering if we should hold off on this PR until then?

Reply:
Yeah I agree :( I should have looked at the API before doing this.